draft-wussler-openpgp-pqc.md

```diff
@@ -830,69 +754,70 @@ The operation `ecdhKem.decap()` is defined as follows:
+Algorithm ID reference | ML-KEM                    | Public key | Secret key | Ciphertext | Key share
+----------------------:| ------------------------- | ---------- | ---------- | ---------- | ---------
+29, 31, 33             | ML&#x2011;KEM&#x2011;768  | 1184       | 2400       | 1088       | 32
+30, 32, 34             | ML&#x2011;KEM&#x2011;1024 | 1568       | 3168       | 1568       | 32
 
-The placeholder `kyberKem` has to be replaced with the specific Kyber-KEM from
-the column "Kyber-KEM" of {{tab-kyber-artifacts}}.
+To instantiate ML-KEM, one must select a parameter set from the column
+"ML-KEM" of {{tab-mlkem-artifacts}}.
 
-The procedure to perform `kyberKem.encap()` is as follows:
+The procedure to perform `ML-KEM.Encaps()` is as follows:
 
- 1. Extract the component public key `kyberPublicKey` that is part of the
+ 1. Extract the encapsulation key `mlkemEncapsKey` that is part of the
     recipient's composite public key
 
- 2. Invoke `(kyberCipherText, keyShare) <- kyberKem.encap(kyberPublicKey)`
+ 2. Invoke `(mlkemCipherText, mlkemKeyShare) <- ML-KEM.Encaps(mlkemEncapsKey)`
 
- 3. Set `kyberCipherText` as the Kyber ciphertext
+ 3. Set `mlkemCipherText` as the ML-KEM ciphertext
 
- 4. Set `keyShare` as the Kyber symmetric key share
+ 4. Set `mlkemKeyShare` as the ML-KEM symmetric key share
 
-The procedure to perform `kyberKem.decap()` is as follows:
+The procedure to perform `ML-KEM.Decaps()` is as follows:
 
- 1. Invoke `keyShare <-  kyberKem.decap(kyberCipherText, kyberPrivateKey)`
+ 1. Invoke `mlkemKeyShare <-  ML-KEM.Decaps(mlkemCipherText, mlkemDecapsKey)`
 
- 2. Set `keyShare` as the Kyber symmetric key
+ 2. Set `mlkemKeyShare` as the ML-KEM symmetric key share
 
-## Composite Encryption Schemes with Kyber {#ecc-kyber}
+## Composite Encryption Schemes with ML-KEM {#ecc-mlkem}
 
-{{kem-alg-specs}} specifies the following Kyber + ECC composite public-key
+{{kem-alg-specs}} specifies the following ML-KEM + ECC composite public-key
 encryption schemes:
 
-{: title="Kyber-ECC-composite Schemes" #tab-kyber-ecc-composite}
-Algorithm ID reference | Kyber-KEM    | ECC-KEM   | ECDH-KEM curve
+{: title="ML-KEM + ECC composite schemes" #tab-mlkem-ecc-composite}
+Algorithm ID reference | ML-KEM       | ECC-KEM   | ECC-KEM curve
 ----------------------:| ------------ | --------- | --------------
-29                     | kyberKem768  | x25519Kem | X25519
-30                     | kyberKem1024 | x448Kem   | X448
-31                     | kyberKem768  | ecdhKem   | NIST P-256
-32                     | kyberKem1024 | ecdhKem   | NIST P-384
-33                     | kyberKem768  | ecdhKem   | brainpoolP256r1
-34                     | kyberKem1024 | ecdhKem   | brainpoolP384r1
-
-The Kyber + ECC composite public-key encryption schemes are built according to
+29                     | ML-KEM-768   | x25519Kem | Curve25519
+30                     | ML-KEM-1024  | x448Kem   | Curve448
```

Comment:
Why changing `X*` to `Curve*`?

Reply:
Because the column lists the curves associated to the algorithm. X* denotes algorithms/functions and Curve* denotes the curves. I felt it should be consistent.